docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Projection/VectorWhiten.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public sealed class VectorWhiten
+    {
+        /// <summary>
+        /// This example require installation of addition nuget package <a href="https://www.nuget.org/packages/Microsoft.ML.HalLearners/">Microsoft.ML.HalLearners</a>
+        /// </summary>
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            var data = SamplesUtils.DatasetUtils.GetVectorOfNumbersData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Features
+            // 0   1   2   3   4   5   6   7   8   9
+            // 1   2   3   4   5   6   7   8   9   0  
+            // 2   3   4   5   6   7   8   9   0   1
+            // 3   4   5   6   7   8   9   0   1   2
+            // 4   5   6   7   8   9   0   1   2   3
+            // 5   6   7   8   9   0   1   2   3   4
+            // 6   7   8   9   0   1   2   3   4   5
+
+            // A small printing utility.
+            Action<string, IEnumerable<VBuffer<float>>> printHelper = (colName, column) =>
+            {
+                Console.WriteLine($"{colName} column obtained post-transformation.");
+                foreach (var row in column)
+                    Console.WriteLine($"{string.Join(" ", row.DenseValues().Select(x => x.ToString("f3")))} ");
+            };
```

Comment:
Can it go to SamplesUtils?

Reply:
Eventually. Not in this PR

---
In reply to: [257356248](https://github.com/dotnet/machinelearning/pull/2497#discussion_r257356248) [](ancestors = 257356248)

Reply:
ok.

---
In reply to: [257356644](https://github.com/dotnet/machinelearning/pull/2497#discussion_r257356644) [](ancestors = 257356644,257356248)